act/fixtures/code_review.yml

```diff
@@ -0,0 +1,25 @@
+name: CDS Code Review with OpenAI
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+permissions: write-all
+jobs:
+  code_review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          path: ./
+      - name: Code Review
+        uses: ./act/fixtures/
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          OPENAI_API_MODEL: ${{ vars.OPENAI_API_MODEL }}
+          OPENAI_API_VERSION: ${{ vars.OPENAI_API_VERSION }}
+          OPENAI_BASE_URL: ${{ vars.OPENAI_BASE_URL }}
+          exclude: "yarn.lock,dist/**"
```

Comment:
Not the real workflow, and suboptimal to store as a duplicate to the real workflow because of the way that `act` needs to be setup and its limitations. Notice how the checkout refers to the local folder and the code review step refers to the fixtures folder itself: this is to debug the action itself from the local code.